### DISCUSSION TITLE
How to get the current caret position in a QEditor?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#11990] [General - Components / Directives / etc]
When typing some text in a QEditor, I concurrently analyze its content and extract some words. The extracted words are removed from the current content.

As an example, I could be looking for any instances of `#andsomething`, and once I see one I would

    note.value.text = note.value.text.replace(`#${tag}`, '')

where `note.value.text` is the content of the QEditor and `tag` is the matched word (`andsomething` in the case above).

This works, but the problem is that the cursor comes back to the beginning of the text upon such replacement. **Is there a good way to keep the cursor where it is?**

I could imagine two ways to do that:
- replace the text in  another way, a way that does not actually replace the whole string, but removes some elements → I do not about such a way, a mutation of the string of sorts
- get the position of the cursor, and restore it after the change → I do not know how to get the current position of the cursor in the QEditor

**Comment by hawkeye64** [comment]
https://stackoverflow.com/a/3976125/5359558
